src/main/scala/net/ruippeixotog/scalascraper/browser/Browser.scala

```diff
@@ -15,6 +15,8 @@ trait Browser {
   def parseFile(path: String, charset: String): Document = parseFile(new File(path), charset)
   def parseFile(path: String): Document = parseFile(new File(path), "UTF-8")
 
+  def parseResource(name: String, charset: String = "UTF-8"): Document = parseFile(new File(getClass.getResource(name).toURI), charset)
```

Comment:
I get kind of stuck on how to implement this for the `HtmlUnitBrowser` - would be interested to see what you have in mind!